src/app/foundation/layout.tsx

```diff
@@ -0,0 +1,44 @@
+"use client";
+
+import React from 'react';
+import startCase from 'lodash/startCase';
+import Head from 'next/head';
+import getFullPageTitle from '@/utils/getFullPageTitle';
+
+import LeafyGreenProvider from '@leafygreen-ui/leafygreen-provider';
+
+import { css } from '@emotion/css';
+
+const DESKTOP_MARGIN_TOP = 72;
+const DESKTOP_MAX_WIDTH = 700;
+
+const layoutStyles = css`
+  margin-top: ${DESKTOP_MARGIN_TOP}px;
+  max-width: ${DESKTOP_MAX_WIDTH}px;
+`;
+
+function ContentPageLayout({
+  contentPageTitle,
+  children,
+}: {
+  contentPageTitle: string;
+  children?: React.ReactNode;
+}) {
+  const pageTitle = getFullPageTitle(startCase(contentPageTitle));
+  return (
+    <div role="main" className={layoutStyles}>
+      <Head>
```

Comment:
do we need to redeclare all of this?

Reply:
ahh I removed this earlier but I think I overwrote my changes to your earlier feedback when I pulled down the latest